src/Uno.Extensions.Reactive.Messaging/MessengerExtensions.cs

```diff
@@ -23,7 +23,10 @@ public static class MessengerExtensions
 	/// <param name="keySelector">A selector to get a unique identifier of a <typeparamref name="TEntity"/>.</param>
 	/// <returns>A disposable that can be used to unbind the state from the messenger.</returns>
 	public static IDisposable Observe<TEntity, TKey>(this IMessenger messenger, IState<TEntity> state, Func<TEntity, TKey> keySelector)
-		=> AttachedProperty.GetOrCreate(state, keySelector, messenger, (s, ks, msg) => new Recipient<IState<TEntity>, TEntity, TKey>(s, msg, ks, StateExtensions.Update));
+		=> AttachedProperty.GetOrCreate(owner: state,
+										key: keySelector,
+										state: messenger,
+										factory: (s, ks, msg) => new Recipient<IState<TEntity>, TEntity, TKey>(s, msg, ks, StateExtensions.Update));
```

Comment:
```suggestion
										factory: static (s, ks, msg) => new Recipient<IState<TEntity>, TEntity, TKey>(s, msg, ks, StateExtensions.Update));
```

The factory delegate should be `static` (as otherwise the cache might get "corrupted"). Even if not required, I try to add the `static` keyword everywhere in extensions to make it clear and avoid mistakes.

BTW about code styling, usually in uno's projects (and in MVUX) we recommend to add only 1 indentation level per new line. Here it should be :

```
		=> AttachedProperty.GetOrCreate(
			owner: state,
			key: keySelector,
			state: messenger,
			factory: static (s, ks, msg) => new Recipient<IState<TEntity>, TEntity, TKey>(s, msg, ks, StateExtensions.Update));
```